docs/guides/settings.rst

```diff
@@ -0,0 +1,21 @@
+Custom settings
+===============
+
+Adjust the image path
++++++++++++++++++++++
+
+By default, the variations are stored under a hashed directory path that by
+default starts with ``dg/``. This setting can be modified in the project's
+settings by setting the ``DAGUERRE_ADJUSTED_IMAGE_PATH`` variable.
+
+Example:
+
+.. code-block:: django
+
+    # settings.py
+    DAGUERRE_ADJUSTED_IMAGE_PATH = 'img'
+
+
+**WARNING:** The maximum length of the ``DAGUERRE_ADJUSTED_IMAGE_PATH`` string
```

Comment:
This should use an rst [admonition](http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions).